backstop/tests/carousel.html

```diff
@@ -504,6 +493,8 @@ <h2>Default</h2>
               break;
           }
         });
+        // initialize
+        updateSlide(0);
```

Comment:
I removed this because it was making our tests unstable. Maybe better just add required classes straight to HTML.

Reply:
I have the classes in the html for the default size. I think this helps in other widths (e.g. mobile)